config.go

```diff
@@ -136,6 +136,12 @@ type autoPilotConfig struct {
 
 type torConfig struct {
 	Socks           string `long:"socks" description:"The port that Tor's exposed SOCKS5 proxy is listening on. Using Tor allows outbound-only connections (listening will be disabled) -- NOTE port must be between 1024 and 65535"`
+	Control         string `long:"control" description:"The port that Tor's ControlPort is listening on -- NOTE port must be between 1024 and 65535"`
+	ControlPassword string `long:"controlpass" description:"The password to be used for authenticating to Tor's ControlPort."`
+	VirtPort        string `long:"virtport" description:"The virtual port as described in Tor's control-spec to be used when creating hidden services -- NOTE port must be between 1024 and 65535"`
+	TargPort        string `long:"targport" description:"The target port as described in Tor's control-spec to be used when creating hidden services -- NOTE port must be between 1024 and 65535"`
+	PrivKey         string `long:"privkey" description:"The private key used to create a hidden service."`
```

Comment:
Perhaps `lnd` should generate this itself? Extending we'd store the private key in a file (with proper permissions), and check if the file is there on start up. If it isn't, we'll create a new onion service with a fresh key. Otherwise we'd use the existing key. We'd have set file names for both v2 and v3. 

This way, rotating onion service identities is as easy as deleting a file.

Reply:
If implemented, the behavior described above means that the `Save` arg isn't needed.